app/presenters/licence_finder_content_item_presenter.rb

```diff
@@ -4,13 +4,17 @@ def base_path
   end
 
   def content_id
-    metadata[:content_id]
+    "69af22e0-da49-4810-9ee4-22b4666ac627"
```

Comment:
Is this change necessary? The content ID should be the same for this content item so this should suffice: https://github.com/alphagov/licence-finder/blob/master/config/initializers/application_metadata.rb#L3